### ISSUE TITLE
verbose output or logging option?

### ISSUE THREAD

**Issue by mlandersbdc** [Nike-Inc/gimme-aws-creds#246]
I'm trying to diagnose an issue where running gimme-aws-creds just hangs with no output in windows 10. There doesn't seem to be a verbose output option or logging option in the documentation of this tool that would point me in the right direction. Does either of these options exist?

**Comment by kellen-t-oconnor** [comment]
Did you ever find a solution? Encountering the same issue.

**Comment by mlandersbdc** [comment]
no. developers using windows run into all kinds of weird problems like this. ideal solution- don't use windows for dev laptops

**Comment by seanorama** [comment]
> no. developers using windows run into all kinds of weird problems like this. ideal solution- don't use windows for dev laptops

That doesn't answer the need for "debug output" or log. I need this on MacOS and Linux.

Using `saml2aws` instead, since it has debug output.
